cts/cts-exec.in

```diff
@@ -226,7 +228,8 @@ class ExecTests(Tests):
             "upstart_cancel_line": '-c cancel -r upstart_test_rsc -a monitor -i 2s ' + self._action_timeout,
             "upstart_cancel_event": '-l "NEW_EVENT event_type:exec_complete rsc_id:upstart_test_rsc action:monitor rc:ok op_status:Cancelled" ',
 
-            "service_reg_line": '-c register_rsc -r service_test_rsc ' + self._action_timeout + ' -C service -T LSBDummy',
+            "service_reg_line": '-c register_rsc -r service_test_rsc ' + self._action_timeout +
+                                ' -C service -T %s' % (service_agent),
```

Comment:
Parens around service_agent are unnecessary here since it's only one item being substituted into the string.